huaweicloud/services/obs/resource_huaweicloud_obs_bucket_acl.go

```diff
@@ -146,37 +146,21 @@ and must be unique.`,
 	return &sc
 }
 
-func buildPermissionTypesFromRawMap(rawMap map[string]interface{}) []obs.PermissionType {
+func buildBucketAccessesFromRawMap(rawMap map[string]interface{}) []string {
 	var accesses []string
 	if accessArray, ok := rawMap["access_to_bucket"].([]interface{}); ok {
 		accesses = append(accesses, utils.ExpandToStringList(accessArray)...)
 	}
 	if accessArray, ok := rawMap["access_to_acl"].([]interface{}); ok {
 		accesses = append(accesses, utils.ExpandToStringList(accessArray)...)
 	}
-
-	permissionTypes := make([]obs.PermissionType, len(accesses))
-	for i, access := range accesses {
-		switch access {
-		case "READ":
-			permissionTypes[i] = obs.PermissionRead
-		case "WRITE":
-			permissionTypes[i] = obs.PermissionWrite
-		case "READ_ACP":
-			permissionTypes[i] = obs.PermissionReadAcp
-		default:
-			permissionTypes[i] = obs.PermissionWriteAcp
-		}
-	}
-	return permissionTypes
+	return accesses
 }
 
-func buildUserTypeGrants(rawMap map[string]interface{}, grants []obs.Grant, domainID string) []obs.Grant {
-	permissionTypes := buildPermissionTypesFromRawMap(rawMap)
-	log.Printf("[DEBUG] The grantee user permission types: %v.", permissionTypes)
-	for _, permissionType := range permissionTypes {
+func buildUserTypeGrants(accesses []string, grants []obs.Grant, domainID string) []obs.Grant {
```

Comment:
I think accountID is more suitable for this function.
```suggestion
func buildUserTypeGrants(accesses []string, grants []obs.Grant, accountID string) []obs.Grant {
```